### PR TITLE
Clearable multi select fields

### LINKED ISSUE #1451: "Clear all" option for participant form field
There should be an option to remove all currently selected participants in the Evaluation form.

### PR DESCRIPTION
**Pull request by JannisBerndt**
I havent had a better idea yet. I am open for some suggestions...

Fixes #1451

### CODE REVIEW

**Comment by JannisBerndt** [comment]
* _I havent had a better idea yet. I am open for some suggestions..._

File: evap/evaluation/templates/base.html [hunk 1]
```diff
@@ -110,7 +110,8 @@
                 language: "{{ LANGUAGE_CODE }}",
                 placeholder: "{% trans 'Please select...' %}",
                 minimumResultsForSearch: 15,
-                minimumInputLength: $(this).children('option').length > 50 ? 3 : 0
+                minimumInputLength: $(this).children('option').length > 50 ? 3 : 0,
+                allowClear: (this.name == "participants") ? true : false
```
Comment (He3lixxx) [inline evap/evaluation/templates/base.html:114]: I'd suggest adding a second argument to this function (`applySelect2`) that allows to provide additional arguments or override some of the default ones. Then, the place where the "exceptional" behaviour is defined is closer to where we actually need it.

(afaik javascript has some object packing and unpacking semantics similar to **kwargs in python)

Comment (janno42) [reply evap/evaluation/templates/base.html:114]: i would replace this line with `allowClear: this.multiple` instead. it's actually helpful to add this for all multi select fields and we don't have to handle it individually.

**Review by janno42 — CHANGES_REQUESTED** [review]

**Comment by janno42** [comment]
Very nice and easy solution 👍

**Review by janno42 — APPROVED** [review]

**Review by He3lixxx — APPROVED** [review]

**Comment by janno42** [comment]
thanks :)

**Pull request merged**